lib/CXGN/BrAPI/v2/Germplasm.pm

```diff
@@ -667,77 +686,100 @@ sub store {
-                    countryOfOriginCode=>$countryOfOriginCode,
-                    typeOfGermplasmStorageCode=>$typeOfGermplasmStorageCode,
-                    donors=>$donors,
-                    acquisitionDate=>$acquisitionDate,
-                    transgenic=>$transgenic,
-                    notes=>$notes,
-                    state=>$state,
-                    variety=>$variety,
-                    locationCode=>$locationCode,
-                    sp_person_id => $user_id,
-                    user_name => $user_name,
-                    modification_note => 'Bulk load of accession information'
+                    schema                          => $schema,
+                    check_name_exists               => 0,
+                    main_production_site_url        => $main_production_site_url,
+                    type                            => 'accession',
+                    type_id                         => $type_id,
+                    species                         => $species,
+                    name                            => $name,
+                    uniquename                      => $uniquename,
+                    organization_name               => $organization_name,
+                    population_name                 => $population_name,
+                    description                     => $description,
+                    accessionNumber                 => $accessionNumber,
+                    germplasmPUI                    => $germplasmPUI,
+                    germplasmSeedSource             => $germplasmSeedSource,
+                    synonyms                        => \@synonymNames,
+                    instituteCode                   => $instituteCode,
+                    instituteName                   => $instituteName,
+                    biologicalStatusOfAccessionCode => $biologicalStatusOfAccessionCode,
+                    countryOfOriginCode             => $countryOfOriginCode,
+                    typeOfGermplasmStorageCode      => $typeOfGermplasmStorageCode,
+                    donors                          => $donors,
+                    acquisitionDate                 => $acquisitionDate eq '' ? undef : $acquisitionDate,
+                    transgenic                      => $transgenic,
+                    notes                           => $notes,
+                    state                           => $state,
+                    variety                         => $variety,
+                    locationCode                    => $locationCode,
+                    sp_person_id                    => $user_id,
+                    user_name                       => $user_name,
+                    modification_note               => 'Bulk load of accession information',
+                    mother_accession                => $mother,
+                    father_accession                => $father,
```

Comment:
BI added the ability to specify pedigree information when creating new germplasm